StreamChatClient_v3/ChatClient.swift

```diff
@@ -379,14 +380,17 @@ extension Client {
             _ requestEncoder: RequestEncoder,
             _ eventDecoder: AnyEventDecoder,
             _ eventMiddlewares: [EventMiddleware]
-        )
-            -> WebSocketClient = {
-                WebSocketClient(connectEndpoint: $0, sessionConfiguration: $1, requestEncoder: $2, eventDecoder: $3,
-                                eventMiddlewares: $4)
-            }
+        ) -> WebSocketClient = {
+            WebSocketClient(connectEndpoint: $0,
+                            sessionConfiguration: $1,
+                            requestEncoder: $2,
+                            eventDecoder: $3,
+                            eventMiddlewares: $4)
+        }
         
-        var databaseContainerBuilder: (_ kind: DatabaseContainer.Kind) throws
-            -> DatabaseContainer = { try DatabaseContainer(kind: $0) }
+        var databaseContainerBuilder: (_ kind: DatabaseContainer.Kind) throws -> DatabaseContainer = {
+            try DatabaseContainer(kind: $0)
+        }
         
```

Comment:
This also should be a separate commit since it has nothing to do with "Replace internal ping timer in WebSocketClient with WebSocketPingController"